crates/bevy_core_pipeline/src/prepass/mod.rs

```diff
@@ -0,0 +1,150 @@
+//! Run a prepass before the main pass to generate depth and/or normals texture, also called a g-buffer.
```

Comment:
Yeah, I've heard people call that a thin g-buffer. I mostly wanted the keyword somewhere so that someone searching for it would find it. I agree that this isn't a full g-buffer.

Reply:
```suggestion
//! Run a prepass before the main pass to generate depth and/or normals texture, sometimes called a thin g-buffer.
```